### ISSUE TITLE
VirtualMachinePool.info() not working

### ISSUE THREAD

**Issue by mpvillerius** [python-oca/python-oca#17]
code :
```python
#!/usr/bin/python
from __future__ import unicode_literals
import oca
# OpenNebula credentials
CREDENTIALS = "x:x"
# XML_RPC endpoint where OpenNebula is listening
ENDPOINT    = "http://ui.hpccloud.surfsara.nl:2633/RPC2"
client = oca.Client(CREDENTIALS, ENDPOINT)
hpc_version=client.version()
print "Surf Sara HPC cloud Version : " + hpc_version
VM_pool = oca.VirtualMachinePool(client)
print "VirtualMachinePool :" + str(VM_pool)
```

```
Prints (vmpool empty):
Surf Sara HPC cloud Version : 4.14.2
VirtualMachinePool :[]
```
Now add the code print VM_pool.info() gives an error:
```
Surf Sara HPC cloud Version : 4.14.2
VirtualMachinePool :[]
root@ubuntu:~/myproject# vi hpc-overview.py 
root@ubuntu:~/myproject# python ./hpc-overview.py 
Surf Sara HPC cloud Version : 4.14.2
VirtualMachinePool :[]
Traceback (most recent call last):
  File "./hpc-overview.py", line 20, in <module>
    print VM_pool.info()
  File "build/bdist.linux-x86_64/egg/oca/vm.py", line 346, in info
  File "build/bdist.linux-x86_64/egg/oca/pool.py", line 111, in info
  File "build/bdist.linux-x86_64/egg/oca/pool.py", line 55, in _initialize_xml
  File "/root/miniconda2/lib/python2.7/xml/etree/ElementTree.py", line 1300, in XML
    parser.feed(text)
  File "/root/miniconda2/lib/python2.7/xml/etree/ElementTree.py", line 1640, in feed
    self._parser.Parse(data, 0)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u200b' in position 3728: ordinal not in range(128)
```
I am on the master branch (oca==4.15.0a1)

Am I doing something wrong?

**Comment by markus-sara** [comment]
No, this is a limitation in the transcoding algorithm. Char '\u200b' is one of the unicode zero-width characters. Maarten-k has submitted pull request 5e20583 that fixes the problem for this particular character occurrence.

**Comment by sysradium** [comment]
Maybe it's better to add and `encode('utf-8')` statement?

**Comment by markus-sara** [comment]
Yes, a more structural solution is needed. Your solution sounds better and when I get round to it I will test if that solves the general problem. Do not hold your breath, though :-)

**Comment by sysradium** [comment]
@markus-sara ok, feel free submit a PR 👍 
I do not have some installed OpenNebula instance, so can't try if it really works, so would be cool to hear from you.

**Comment by sysradium** [comment]
Closed in favour #27
